test/options-validator.test.js

```diff
@@ -0,0 +1,52 @@
+const {
+    optionsValidationHandler,
+} = require('../src/handlers/options-validation');
+
+describe('Options validation', () => {
+    let nextHolder;
+    let options;
+
+    beforeEach(() => {
+        nextHolder = {
+            next() { },
+        };
```

Comment:
Is the only purpose of this to have an object with a dummy `next` function? It took me a while to figure it out. Btw, why not create a `const nextMock = () => {}` and pass it everywhere? What's the purpose of the object? It's all a bit confusing.